### PR TITLE
[11.0] [MIG] Migrate account_move_line_tax_editable

### PR DESCRIPTION
**Pull request by feketemihai**
Migrate account_move_line_tax_editable to 11.0

### CODE REVIEW

**Comment by feketemihai** [comment]
@jholze

File: account_move_line_tax_editable/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Account Move Line Tax Editable',
     'summary': """
         Allows to edit taxes on non-posted account move lines""",
-    'version': '10.0.1.0.0',
+    'version': '1.0.1.0.0',
```
Comment (pedrobaeza) [inline account_move_line_tax_editable/__manifest__.py]: 11

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by jholze** [comment]
@feketemihai 🥇

**Review by pedrobaeza — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@feketemihai If you squash your last 2 commits I can merge now.

**Comment by feketemihai** [comment]
@jbeficent Squashed...Thanks.

**Comment by JordiBForgeFlow** [comment]
@feketemihai Thanks!!!

**Pull request merged**